contrib/platform/src/com/sun/jna/platform/win32/Oleacc.java

```diff
@@ -0,0 +1,205 @@
+     *             IID_IDispatch, but it can also be IID_IUnknown, or the IID of any interface that the object is expected to support.
+     * @param ppvObject [out] Address of a pointer variable that receives the address of the specified interface.
+     * @return the HRESULT. If successful, returns S_OK. If not successful, returns one of the error values, or another standard COM error code.
+     */
+    HRESULT AccessibleObjectFromWindow(HWND hwnd, DWORD dwId, REFIID riid, PointerByReference ppvObject);
+
+    /**
+     * Retrieves the window handle that corresponds to a particular instance of an IAccessible interface.
+     *
+     * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/oleacc/nf-oleacc-windowfromaccessibleobject">WindowFromAccessibleObject function (oleacc.h)</a>
+     *
+     * @param IAccessible [in] Pointer to the IAccessible interface whose corresponding window handle will be retrieved. This parameter must not be NULL.
+     * @param phwnd [out] Address of a variable that receives a handle to the window containing the object specified in pacc.
+     *              If this value is NULL after the call, the object is not contained within a window;
+     *              for example, the mouse pointer is not contained within a window.
+     * @return the HRESULT. If successful, returns S_OK. If not successful, returns one of the error values, or another standard COM error code.
+     */
+    HRESULT WindowFromAccessibleObject(Pointer IAccessible, PointerByReference phwnd);
+
+    /**
+     * Retrieves the localized string that describes the object's role for the specified role value.
+     *
+     * @see <a href="https://docs.microsoft.com/en-us/windows/win32/api/oleacc/nf-oleacc-getroletexta">GetRoleTextA function (oleacc.h)</a>
+     *
+     * @param lRole [in] One of the object role constants.
+     * @param lpszRole [out] Address of a buffer that receives the role text string. If this parameter is NULL, the function returns the role string's length, not including the null character.
+     * @param cchRoleMax [in] The size of the buffer that is pointed to by the lpszRole parameter. For ANSI strings, this value is measured in bytes; for Unicode strings, it is measured in characters.
+     * @return If successful, and if lpszRole is non-NULL, the return value is the number of bytes (ANSI strings)
+     *         or characters (Unicode strings) copied into the buffer, not including the terminating null character.
+     *         If lpszRole is NULL, the return value represents the string's length, not including the null character.
+     *         If the string resource does not exist, or if the lpszRole parameter is not a valid pointer, the return value is zero (0).
+     *         To get extended error information, call GetLastError.
+     */
+    UINT GetRoleTextA(DWORD lRole, LPSTR lpszRole, UINT cchRoleMax);
```

Comment:
Generally preferred to just map `int` rather than `DWORD` and `UINT`.  Lots of other instances, just noted this one.

Reply:
Awesome, I've changed all `DWORD` and `UINT` to map to `int` instead.